R/general_lineplot.R

```diff
@@ -139,5 +145,11 @@ general_lineplot <- function(
                     label = c(0.001, 0.01, 0.1, 1, 10, 100, 1000)) +
       labs(y = paste0("Log 10 - ", plt$labels$y))
   }
+
+  if (show_threshold == TRUE) {
```

Comment:
issue(non-blocking): Minor thing, but:
```suggestion
  if (show_threshold) {
```